src/createServer.js

```diff
@@ -1,13 +1,67 @@
 'use strict';
 
-// const express = require('express');
-
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const express = require('express');
+  const fs = require('fs');
+  const path = require('path');
+
+  // Paths:
+  const publicDirPath = path.join(__dirname, 'public');
+  // Port:
+  // const PORT = process.env.PORT || 5000;
+  // App setup:
+  const app = express();
+  const usersController = require('./usersAPI/users-controller');
+  const expensesController = require('./expensesAPI/expenses-controller');
+
+  // Middleware:
+  app.use(express.json());
+
+  // ======= API Home page:
+  app.get('/', (req, res) => {
+    const indexHTMLPath = path.join(publicDirPath, 'index.html');
+
+    res.end(fs.readFileSync(indexHTMLPath));
+  });
+
+  // ======= USERS API:
+  // GET ALL:
+  app.get('/users', usersController.getAllUsers);
+
+  // GET ONE:
+  app.get('/users/:userID', usersController.getOneUsers);
+
+  // POST ONE:
+  app.post('/users', usersController.createOneUser);
+
+  // PATCH ONE:
+  app.patch('/users/:userID', usersController.updateUser);
+
+  // DELETE ONE:
+  app.delete('/users/:userID', usersController.deleteUser);
+
+  // ======= EXPENSES API:
```

Comment:
same about expenses

Reply:
in this way your createServer function will be more general, and will contain only the most important functions without any other calls, related to specific model